lib/model/recent_dm_conversations.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:collection/collection.dart';
+import 'package:flutter/foundation.dart';
+
+import '../api/model/initial_snapshot.dart';
+import '../api/model/model.dart';
+import '../api/model/events.dart';
+import 'narrow.dart';
+
+/// A view-model for the recent DM conversations UI.
```

Comment:
nit: I think this parses better with some hyphens in that attributive compound noun:
```suggestion
/// A view-model for the recent-DM-conversations UI.
```